src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/Core/IdeaCollection/Poll/ProposalColumns.html

```diff
@@ -0,0 +1 @@
+<adh-poll-detail-column data-transclusion-id="transclusionId" data-view="view" data-process-properties="processProperties"></adh-poll-detail-column>
```

Comment:
One may ask: Why isn't this a detail column already? The answer is: We need to get `proposalUrl` into the scope, and for that we need a separate directive.